java-frontend/src/main/java/org/sonar/plugins/java/api/JavaVersion.java

```diff
@@ -96,4 +96,6 @@
    * @return empty string if java version is properly set, default message otherwise.
    */
   String java8CompatibilityMessage();
+
+  boolean isJava9Compatible();
```

Comment:
We should put this method next to the other similar one (line 53), with proper javadoc since it will be part of the public API.